pkg/webhook/policy.go

```diff
@@ -68,7 +69,10 @@ import (
 // https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#response
 const httpStatusWarning = 299
 
-var maxServingThreads = flag.Int("max-serving-threads", -1, "cap the number of threads handling non-trivial requests, -1 caps the number of threads to GOMAXPROCS. Defaults to -1.")
+var (
+	validateTemplateRego = flag.Bool("validate-template-rego", true, "validate rego code for constraints template. Defaults to true. To be removed before 3.16. Not to be used with `experimental-enable-k8s-native-validation` is set to true.")
```

Comment:
s/Not to be used with/Not to be used when/

Reply:
```suggestion
	validateTemplateRego = flag.Bool("validate-template-rego", true, "validate rego code for constraints template. Defaults to true. To be removed in v3.16. Use gator to validate in shift left manner to avoid impact with this behavior change. Not to be used when `experimental-enable-k8s-native-validation` is set to true.")
```